test/presenters/contact_presenter_test.rb

```diff
@@ -88,8 +88,8 @@ def schema_name
       schema = schema_item("contact_with_webchat")
       presented = present_example(schema)
       assert_equal true, presented.show_webchat?
-      assert_equal presented.webchat.availability_url, "https://www.tax.service.gov.uk/csp-partials/availability/1030"
-      assert_equal presented.webchat.open_url, "https://www.tax.service.gov.uk/csp-partials/open/1030"
+      assert_equal presented.webchat.availability_url, "https://hmrc-uk.digital.nuance.com/tagserver/launch/agentAvailability?agentGroupID=10006852&siteID=10006719&businessUnitID=19001235"
+      # assert_equal presented.webchat.open_url, "https://tax.service.gov.uk/ask-hmrc/webchat/child-benefit"
```

Comment:
Why is this one commented? Can we update the assertion instead of removing it?

Reply:
fixed the assert to equal correct value